### PR TITLE
Feature modal request change

### PR DESCRIPTION
**Pull request by AngieCristina**
**Pair programming with Marieth**
- Added modals request
- Created a calendar component
- Created request service to connect Api with Frontend
- Amended data

### CODE REVIEW

File: App/src/views/RequestsView.js [hunk 1]
```diff
@@ -1,17 +1,44 @@
...
+import React, { useState, useEffect } from "react";
 import { jsx } from "@emotion/core";
 import Nabvar from "../components/Nabvar";
 import { Redirect } from "@reach/router";
 import { useUser } from "../contexts/user";
+import RequestModal from "./RequestModal";
+import requestsFetch from "../services/request";
+import { users } from "../services/user";
 
 const request = [
```
Comment (MariethPerez) [inline App/src/views/RequestsView.js]: I think , This lines could be removed.

File: App/src/views/RequestsView.js [hunk 2]
```diff
@@ -1,17 +1,44 @@
...
+  const [frontdesks, setFrontdesks] = useState([]);
   const user = useUser();
+  useEffect(() => {
+    async function fetchData() {
+      const response = await requestsFetch.requestsFetch();
+      setRequests(response);
+    }
+    fetchData();
+  }, []);
+  console.log(requests);
```
Comment (MariethPerez) [inline App/src/views/RequestsView.js]: I think , This line could be removed.

File: App/src/views/RequestsView.js [hunk 3]
```diff
@@ -24,7 +51,8 @@ function RequestsView() {
           margin: "0 auto"
         }}
       >
-        {request.map(request => (
+        {/* {id: 5, creationDate: null, requester_id: 7, requested_id: 8, rol: "FrontDesk", …} */}
```
Comment (MariethPerez) [inline App/src/views/RequestsView.js]: I think , This lines could be removed.

**Review by MariethPerez — APPROVED** [review]

File: App/src/components/Calendar.js [hunk 4]
```diff
@@ -0,0 +1,289 @@
...
+
+  useEffect(() => {
+    async function fetchData() {
+      const response = await schedules.schedules();
+      setEvents(response);
+    }
+    fetchData();
+  }, []);
+
+  console.log(modalIsOpen);
```
Comment (yummta) [inline App/src/components/Calendar.js:109]: Remember clean your `console.log`

**Review by yummta — APPROVED** [review]

**Pull request merged**